internal/uvm/create.go

```diff
@@ -315,3 +316,15 @@ func (uvm *UtilityVM) normalizeMemorySize(ctx context.Context, requested int32)
 func (uvm *UtilityVM) DevicesPhysicallyBacked() bool {
 	return uvm.devicesPhysicallyBacked
 }
+
+// Closes the external GCS connection if it is being used and also closes the
+// listener for GCS connection.
+func (uvm *UtilityVM) CloseGCSConnection() (err error) {
+	if uvm.gc != nil {
+		err = uvm.gc.Close()
+	}
+	if err != nil && uvm.gcListener != nil {
```

Comment:
Why do we need this extra err check that we didn't previously have?

Reply:
You are right. I must have added it as a habit of checking the previous error. Fixed it.